Source/Web View/WebViewController.swift

```diff
@@ -346,6 +349,18 @@ extension WebViewController {
         
         return false
     }
+
+    // MARK: - NSURLSessionDelegate
+    public func URLSession(session: NSURLSession, didReceiveChallenge challenge: NSURLAuthenticationChallenge, completionHandler: (NSURLSessionAuthChallengeDisposition, NSURLCredential!) -> Void) {
+        if challenge.protectionSpace.authenticationMethod == NSURLAuthenticationMethodServerTrust {
+            if let host = self.challengeHost {
+                if challenge.protectionSpace.host == host {
```

Comment:
just a suggestion but you could avoid the extra nested `if` by using a where clause after the if let binding

```
if let host = challengeHost 
	where challenge.protectionSpace.host == host {
	    let credential = NSURLCredential(forTrust: challenge.protectionSpace.serverTrust)
	    completionHandler(NSURLSessionAuthChallengeDisposition.UseCredential, credential)
}
```